src/Objects/Keyboard/ReplyMarkupKeyboard.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Telegram\Bot\Objects\Keyboard;
+
+/**
+ * Class ReplyKeyboardMarkup
+ *
+ * @method $this setResizeKeyboard($boolean)     (Optional). Requests clients to resize the keyboard vertically for optimal fit.
+ * @method $this setOneTimeKeyboard($boolean)    (Optional). Requests clients to hide the keyboard as soon as it's been used.
+ * @method $this setSelective($boolean)          (Optional). Use this parameter if you want to show the keyboard to specific users only.
+ */
+class ReplyMarkupKeyboard extends BaseKeyboard
+{
+    /**
+     * Make the keyboard with given data.
+     *<p></p>
+     *
+     * Alternatively you can pass empty data and then use fluent syntax to build the keyboard up.
+     *
+     * <code>
+     * [
+     *   'keyboard'          => '',  //  KeyboardButton[] - Array of button rows, each represented by an Array of KeyboardButton objects
+     *   'resize_keyboard'   => '',  //  bool             - (Optional). Requests clients to resize the keyboard vertically for optimal fit (e.g., make the keyboard smaller if there are just two rows of buttons). Defaults to false, in which case the custom keyboard is always of the same height as the app's standard keyboard.
+     *   'one_time_keyboard' => '',  //  bool             - (Optional). Requests clients to hide the keyboard as soon as it's been used. The keyboard will still be available, but clients will automatically display the usual letter-keyboard in the chat – the user can press a special button in the input field to see the custom keyboard again. Defaults to false.
+     *   'selective'         => '',  //  bool             - (Optional). Use this parameter if you want to show the keyboard to specific users only. Targets: 1) users that are @mentioned in the text of the Message object; 2) if the bot's message is a reply (has reply_to_message_id), sender of the original message.
+     * ]
+     * </code>
+     *
+     * @link https://core.telegram.org/bots/api#replykeyboardmarkup
+     *
+     * @param mixed $data
+     *
+     * @return static
+     */
+    public static function create($data = []): self
+    {
+        return new static($data);
+    }
+
+//    /**
+//     * Button Label Text. (Required).
+//     *
+//     * Use this method if you wish to use fluent syntax to create the keyboard.
+//     *
+//     * @param array $buttons
+//     *
+//     * @return ReplyMarkupKeyboard
+//     */
+//    public static function keyboard(array $buttons = []): self
+//    {
```

Comment:
What's with these commented out code? Maybe remove it if not needed.

Reply:
Removed.